guide/glossary/node.md

```diff
@@ -0,0 +1,219 @@
+
+## Common node setups
+
+From complete control to fully trusting a single third party, there are a variety of node setups. Below are examples of common ones.
+
+#### Local node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/local-node.png"
+   retina = "/assets/images/guide/glossary/node/local-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+A very simple setup is to use an application like Bitcoin Core that combines node and wallet functionality.
+
+</div>
+
+#### Hardware node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/hardware-node.png"
+   retina = "/assets/images/guide/glossary/node/hardware-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+Separating node and wallet functionality, many users rely on a dedicated computer like a Raspberry Pi running software like Umbrel, MyNode, and Raspiblitz. Wallet applications can connect to the node which allows them to be more lightweight.
+
+</div>
+
+#### Self-hosted node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/own-hosted-node.png"
+   retina = "/assets/images/guide/glossary/node/own-hosted-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+While it possible to run a node on a rented server, this partially places the node in the control of a third party and introduces privacy risks.
+
+</div>
+
+#### Multiple third-party nodes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/multiple-third-party-nodes.png"
+   retina = "/assets/images/guide/glossary/node/multiple-third-party-nodes@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+Connecting to multiple untrusted third-parties can be a good idea if compact block filters are used. This prevents third parties from analyzing which data belongs to the user, an important privacy benefit. It also reduces the risk of receiving manipulated data from a single third-party node, as data from multiple sources can be compared.
+
+</div>
+
+#### Third-party node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/third-party-node.png"
+   retina = "/assets/images/guide/glossary/node/third-party-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+Connecting to a single node controlled by a third-party is a convenient setup for starters, or when the economic cost of running a node is too high. This should only be used if there is a good trust relationship or potential loss is minimal. Too many user relying on third-party nodes can lead to centralization, so it is generally discouraged.
```

Comment:
looks good. only thing I'd ask is what does a potential loss look like for on-chain third party node? besides leaking some identifying information?

Reply:
Good question. They could block your transactions and feed you with fake data to mess with you. Not sure if that power could then be used to get to your funds...

Reply:
So an inconvenience. Loss of time not loss of funds.